sssd_test_framework/hosts/base.py

```diff
@@ -296,3 +296,69 @@ def __init__(self, *args, **kwargs) -> None:
 
         self.fs: LinuxFileSystem = LinuxFileSystem(self)
         self.svc: SystemdServices = SystemdServices(self)
+        self._os_release: dict = {}
+        self._distro_name: str | None = None
+        self._distro_major: int | None = None
+        self._distro_minor: int | None = None
```

Comment:
It might be nicer to initialize here to some default values and move the call to `_distro_information` into `pytest_setup` and remove the `@property`ies. It would simplify the code. But I don't insist, whatever you prefer.

Reply:
It would be run only once, not for each test.